### ISSUE TITLE
Ag grid accessibility failing on clickable rating icon 

### ISSUE THREAD

**Issue by ChithiraMG** [ag-grid/ag-grid#7280]
**I'm submitting a ...**  (check one with "x")
```
[] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[x] support request => see 'Requesting Community Support'
```

**Requesting Community Support**

**Current behaviour.**
I have added a fluent ui  rating icon in ag grid as shown below
![image](https://github.com/ag-grid/ag-grid/assets/101738825/6f6edf67-4265-4805-88db-ef13d436dc61)
fluent ui rating icon link: https://developer.microsoft.com/en-us/fluentui#/controls/web/rating

While doing accessibility check its showing accessibility issue as below
![image](https://github.com/ag-grid/ag-grid/assets/101738825/8d736148-02b3-45c2-9022-ad8008e1bfc9)
![image](https://github.com/ag-grid/ag-grid/assets/101738825/72a896e5-4f47-46c2-90d5-baa1cb084808)

```
 <Rating
                    size={RatingSize.Large}
                    rating={props.node.data ? props.node.data.isFavourite : props.node.allLeafChildren[0].data.isFavourite} readOnly min={0} max={1} allowZeroStars onClick={(e) => { e.stopPropagation(); handleFavouriteOnClick(props.data) }}
                />
```

Tried adding aria-label / aria-labelledby/ title still issue is not solved

* **AG Grid version:** X.X.X
  "ag-charts-react": "^7.2.0",
    "ag-grid-community": "^30.0.0",
    "ag-grid-enterprise": "^30.0.0",
    "ag-grid-react": "^30.0.0",

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]

**Comment by ChithiraMG** [comment]
Can you please tell me if there is any update on this ticket?

**Comment by gportela85** [comment]
The `div` mentioned above is not part of Ag Grid's structure. I believe this problem should be reported in the `fluentui` repository.

However, I have created this example with the Rating component and I don't see any warnings/errors while scanning accessibility checks - https://codesandbox.io/p/sandbox/ag-grid-modules-forked-hghrtv
